syncstorage-mysql/src/pool.rs

```diff
@@ -99,6 +99,13 @@ impl MysqlDbPool {
         })
     }
 
+    /// Spawn a task to periodically evict idle connections. Calls wrapper sweeper fn
+    ///  to use pool.retain, retaining objects only if they are shorter in duration than
+    ///  defined max_idle. Noop for mysql impl.
+    pub fn spawn_sweeper(&self, _interval: Duration) {
```

Comment:
If this is a function that all implementations of `DbPool` should perform, we should add it to the `DbPool` trait (and move this to the `impl DbPool` block.

I wonder if we should rename this to `spawn_maintenance` (`spawn_janitor`?) 

While we don't need to sweep for pool connections for SQL libs, folk may want to do other maintenance things like run `vacuum` or the like.